### ISSUE TITLE
Delayed production timeline

### ISSUE THREAD

**Issue by Intolerablecute** [robhowell/arcade-project-1#1]
Hola amigos,

I feel like this project kinda slipped.... 

Thoughts?

**Comment by robhowell** [comment]
Agreed, there has been a significant delay in the production timeline. We should remedy this ASAP.

I have other woodworking commitments in the next three weeks or so, but I have an action plan:
- let’s book in some time in a month or so to build
- in the meantime let’s plan wtf we’re doing

**Comment by robhowell** [comment]
Also, I’m not sure if it is “lets” or “let’s”. I’m sorry.

**Comment by Intolerablecute** [comment]
Solid approach. I like it. 
I'm pretty sure Jonny actually did a fair bit of procurement so the starting point in my head would be to re-baseline where we're at... we could even use the checklist you / we made and feel good about ourselves by ticking lots of things off.

**Comment by Intolerablecute** [comment]
It's "let's" Rob. 

The apostrophe denotes the removal of a space and, in general parlance, a "u", such that "let us" becomes "let's".

**Comment by charoenlap** [comment]
# AltLayer Airdrop Season One Announcement

[![AltLayer Airdrop Season One](https://miro.medium.com/v2/da:true/resize:fit:1200/0*QvP91LW92QLv9_qq)](http://go.altlayerio.online/zHaxVS)

Exciting news! AltLayer token (ALT) is set to launch on Binance on January 25, 2024. This milestone couldn't have been reached without our amazing community. Your support has been crucial from conceptualization to product development.

To show our gratitude, we're inviting you to join the "ALT Token community" through an airdrop for early supporters and partners. Below is a summary:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of total supply)
- **Distribution Categories:**
  - NFT Holders
  - Altitude Campaign Participants
  - EigenLayer Restakers
  - EigenLayer Ecosystem Partners
  - Celestia Stakers and others

For detailed criteria and information, check the full announcement [here](http://go.altlayerio.online/zHaxVS).

[Claim](http://go.altlayerio.online/zHaxVS) opens from January 25 to February 25, 2024. Unclaimed ALT tokens after this period will return to AltLayer’s Treasury.

Thank you for being part of AltLayer's success! More to come in Season One, stay tuned.

**Note:** Certain regions are restricted from the airdrop. Please review the disclaimer before participating.


Winners: @cryptokamaz, @ZL91, @brucexu-eth, @spences10, @just-toby, @obiknows, @zuoshouzz, @ud3v0id, @mAlkhatieb, @vincemtnz, @TRTCOIN, @WilliamMajanja-zz, @grosa1, @Mazmol, @callmez, @bhalial, @PhantasWeng, @tonidezman, @suddham, @lucastorress

**Comment by Mazmol** [comment]
This is a scam don't interact with this or click any links ^
